packages/prettier-plugin/src/printer.ts

```diff
@@ -306,7 +306,7 @@ export function createSquigglePrinter(
             "}",
           ]);
         case "String":
-          return ['"', node.value, '"'];
+          return [JSON.stringify(node.value)];
```

Comment:
Was this meant as this PR?

Reply:
This code is already merged; I have a theory on why it's in a diff, but you're right that it shouldn't matter, so I won't investigate.

Reply:
Makes sense, I figured that was likely.